paper.md

```diff
@@ -0,0 +1,178 @@
+| The setup and installation instructions for the lesson were complete and easy to follow.   | 0                             | 0               | 4                | 13           | 21                    | 38           | 4,4                     |
+| Examples and tasks in the lesson were relevant and authentic                               | 0                             | 0               | 5                | 19           | 14                    | 38           | 4,2                     |
+Table 1: Agreement on statements by students from 2 workshops taught at the Netherlands eScience Center. 
+The results from these 2 workshops are a good representation of the general feedback we get when teaching this workshop.
+
+|                                                                           |          POOR    |     FAIR    |     GOOD    |     VERY GOOD    |     EXCELLENT    |     N/A    |     TOTAL    |     WEIGHTED AVERAGE    |
+|---------------------------------------------------------------------------|------------------|-------------|-------------|------------------|------------------|------------|--------------|-------------------------|
+|     Introduction into Deep Learning                                       | 0 (0%)           | 2 (5%)      | 10 (27%)    | 8 (22%)          | 17 (46%)         | 0 (0%)     | 37           | 4,1                     |
+|     Classification by a Neural Network using Keras (penguins dataset)     | 0 (0%)           | 1 (3%)      | 5 (13%)     | 16 (42%)         | 16 (42%)         | 0 (0%)     | 38           | 4,2                     |
+|     Monitoring and Troubleshooting the learning process (weather dataset) | 0 (0%)           | 0 (0%)      | 4 (11%)     | 18 (47%)         | 16 (42%)         | 0 (0%)     | 38           | 4,3                     |
+|     Advanced layer types (CIFAR-10 dataset)                               | 0 (0%)           | 2 (5%)      | 5 (13%)     | 7 (18%)          | 16 (42%)         | 8 (21%)    | 38           | 4,2                     |
+Table 2: Quality of the different episodes of the workshop as rated by students from 2 workshops taught at the Netherlands eScience Center. 
+The results from these 2 workshops are a good representation of the general feedback we get when teaching this workshop.
+
+# Citations
+
+Citations to entries in paper.bib should be in
+[rMarkdown](http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html)
+format.
+
+For a quick reference, the following citation commands can be used:
+- `@author:2001`  ->  "Author et al. (2001)"
+- `[@author:2001]` -> "(Author et al., 2001)"
+- `[@author1:2001; @author2:2001]` -> "(Author1 et al., 2001; Author2 et al., 2002)"
+
+# Figures
+
+Figures can be included like this: ![Example figure.](figure.png)
+
```

Comment:
I propose a concluding section where we could discuss how and where the lesson might fit in a learning pathway, e.g. what could learners move onto next after following this lesson? What could instructors teach them beforehand, to ensure they are ready to follow this lesson?

Reply:
Good point. It ends quite abruptly currently. Implemented with c65c67188291083f6581fb479d1a6f67a07b1e6d